src/tests/SimTest.cpp

```diff
@@ -24,6 +35,21 @@ const std::string skokloster =
     Cr::Utility::Directory::join(SCENE_DATASETS,
                                  "habitat-test-scenes/skokloster-castle.glb");
 
+// TODO(MM): move following to common test utils module
+template <typename T, typename U>
+bool equalWithTolerance(const T& lhs, const T& rhs, U tolerance) {
+  return std::abs(rhs - lhs) <= std::abs(tolerance);
+}
+
+bool pixelEqualWithChannelTolerance(const Mn::Color4ub& lhs,
+                                    const Mn::Color4ub& rhs,
+                                    int tolerance) {
+  return equalWithTolerance(lhs.r(), rhs.r(), tolerance) &&
+         equalWithTolerance(lhs.g(), rhs.g(), tolerance) &&
+         equalWithTolerance(lhs.b(), rhs.b(), tolerance) &&
+         equalWithTolerance(lhs.a(), rhs.a(), tolerance);
```

Comment:
As I mentioned in my other comment, this is another thing Corrade's TestSuite has builtin -- basically the following, plus you get a clear message if the comparison fails (as opposed to "false is not true"):

```cpp
#include <Corrade/TestSuite/Tester/Compare/Numeric.h>

CORRADE_COMPARE_WITH(lhs, rhs, TestSuite::Compare::around(tolerance));
```

Though this is also related to the image comparison, and there the fuzzy compare is handled in an even more convenient way :)